.github/tests/images.json

```diff
@@ -0,0 +1,46 @@
+{
+  "spire-server/values.yaml": [
+    {
+      "query": "tests.bash.image",
+      "filter": "^latest-",
```

Comment:
We switched to use the semver tag here. So this also abstracts away the need of daily patching.

Reply:
same